qa-tests-backend/src/test/groovy/BaseSpecification.groovy

```diff
@@ -422,6 +423,34 @@ class BaseSpecification extends Specification {
         orchestrator.deleteSecret("gcr-image-pull-secret", Constants.ORCHESTRATOR_NAMESPACE)
     }
 
+    static addRedHatImagePullSecret(ns = Constants.ORCHESTRATOR_NAMESPACE) {
+        if (!Env.IN_CI && (Env.get("REDHAT_USERNAME", null) == null ||
+                           Env.get("REDHAT_PASSWORD", null) == null)) {
+            LOG.warn "The REDHAT_USERNAME and/or REDHAT_PASSWORD env var is missing. " +
+                    "(this is ok if your test does not use images from registry.redhat.io)"
+            return
+        }
+
+        Kubernetes orchestrator = OrchestratorType.create(
+                Env.mustGetOrchestratorType(),
+                ns
+        )
```

Comment:
maybe not a time for a refactor but how about passing orchestrator instead of ns as it's created just before this method call?

Reply:
https://github.com/stackrox/stackrox/pull/16573

I haven't removed the `ns` parameter because then even if it would be OK in practice we'd be changing the contract of those functions (user can specify which namespace to create the secret in).